bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/ImageDataModel.java

```diff
@@ -31,7 +31,7 @@
      *
      * @since com.adobe.cq.wcm.core.components.models.datalayer 1.0.0
      */
-    @JsonProperty("asset")
+    @JsonProperty("image")
     default AssetDataModel getImage() {
```

Comment:
I'd call this method `getAssetData()`

Reply:
Done in 2d1846aedbd65f6c9483a948382dc9ee13d9588b